src/kvirc/kernel/KviOptions.cpp

```diff
@@ -341,6 +341,9 @@ KviBoolOption g_boolOptionsTable[KVI_NUM_BOOL_OPTIONS]=
 	BOOL_OPTION("MenuBarVisible",true,KviOption_sectFlagFrame | KviOption_resetUpdateGui),
 	BOOL_OPTION("WarnAboutHidingMenuBar",true,KviOption_sectFlagFrame),
 	BOOL_OPTION("WhoRepliesToActiveWindow",false,KviOption_sectFlagConnection),
+#if defined(COMPILE_ON_WINDOWS) || defined(COMPILE_ON_MINGW)
+	BOOL_OPTION("ShowTaskBarButton",true,KviOption_sectFlagFrame),
```

Comment:
you could just remove the `#if defined` since the option will only be able to be used via other preprocessor guards and make a comment next to it saying it's windows specific.